transaction-status/src/parse_accounts.rs

```diff
@@ -13,7 +13,7 @@ pub fn parse_accounts(message: &Message) -> Vec<ParsedAccount> {
     for (i, account_key) in message.account_keys.iter().enumerate() {
         accounts.push(ParsedAccount {
             pubkey: account_key.to_string(),
-            writable: message.is_writable(i),
+            writable: message.is_writable(i, /*demote_sysvar_write_locks=*/ true),
```

Comment:
Note: The other of the 2 hardcoded places. This one is used from `ledger-tool/src/bigtable.rs`, `core/src/rpc.rs`, `cli/src/cli.rs` and possibly elsewhere.

Reply:
@jstarry maybe, this means this recently-added useful stats on explorer (https://explorer.solana.com/block/71320343/accounts) starts to reflect the demoted definition as soon as deployed without feature gate? I think it's ok, just checking :)